src/Features/VisualBasic/Portable/CodeRefactorings/SyncNamespace/VisualBasicChangeNamespaceService.vb

```diff
@@ -36,17 +38,32 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ChangeNamespace
             End If
         End Function
 
-        ' This is only reachable when called from a VB refacoring provider, which is not implemented yet.
+        ' TODO: Implement the service for VB
```

Comment:
if we haven't done this feature for VB, why do we need this service at all?  Why not just remove it entirely?

Reply:
Need to call `TryGetReplacementReferenceSyntax`, so we can fix references in VB.